ShopperInsights/src/main/java/com/braintreepayments/api/ShoppingInsightsApiCall.kt

```diff
@@ -0,0 +1,32 @@
+package com.braintreepayments.api
+
+import PaymentMethodDetails
+import PaymentMethods
+import ShopperInsightApiResult
+
+internal class ShoppingInsightsApiCall(
```

Comment:
Take it or leave it comment - since the endpoint for this API is called `v2/payments/find-eligible-methods` maybe renaming to FindEligiblePaymentsAPI will be easier, since it's more 1:1 with the underlying API

Reply:
My only concern is that we break the naming consistency we have for everything else called Shopping Insights

Reply:
I'm good with Sammy's suggestion, if leaving it as is, let's drop the `Call` from the name since we don't use that elsewhere. It looks like we aren't entirely consistent throughout the SDK but we use either `PaymentFeatureApi` or `PaymentFeatureInternalClient` for the classes that handle making API calls

Reply:
I am cool with dropping the Call name part. fixed